commands/cloudapi-v6/server.go

```diff
@@ -1415,3 +1901,24 @@ func DeleteAllServers(c *core.CommandConfig) error {
 	fmt.Fprintf(c.Command.Command.OutOrStdout(), jsontabwriter.GenerateLogOutput("Servers successfully deleted"))
 	return nil
 }
+
+func DefaultCpuFamily(c *core.CommandConfig) (string, error) {
+	dcId := viper.GetString(core.GetFlagName(c.NS, cloudapiv6.ArgDataCenterId))
+
+	dc, _, err := client.Must().CloudClient.DataCentersApi.DatacentersFindById(context.Background(), dcId).Execute()
+	if err != nil {
+		return "", err
+	}
+
+	if dc.Properties.CpuArchitecture == nil {
+		return "", errors.New("could not retrieve CpuArchitecture")
+	}
+
+	cpuArch := (*dc.Properties.CpuArchitecture)[0]
+
+	if cpuArch.CpuFamily == nil {
+		return "", errors.New("could not retrieve CpuFamily")
+	}
+
+	return *cpuArch.CpuFamily, nil
+}
```

Comment:
Looks great. 

A nitpick,  would be also good if properties had a nil-guard. But this is fine.